src/examples/continue_pipeline.yml

```diff
@@ -5,9 +5,9 @@ usage:
   version: 2.1
   setup: true
   orbs:
-    continuation: circleci/continuation@0.1
+    continuation: circleci/continuation@0.5.0
   workflows:
     use-my-orb:
       jobs:
         - continuation/continue:
-            configuration_path: "./configB.yml"
+            configuration_path: ".circleci/continue_config.yml"
```

Comment:
Do you know if the continue config has to be in the `.circleci` folder?

Reply:
It doesn't, but the previous example would make it look for `configB.yml` in the root folder, which I don't think it's intuitive.